replica/replica.go

```diff
@@ -883,16 +880,15 @@ func (r *Replica) rmDisk(name string) error {
 		return nil
 	}
 
-	lastErr := os.Remove(r.diskPath(name))
-	if err := os.Remove(r.diskPath(name + metadataSuffix)); err != nil {
-		lastErr = err
+	if err := os.Remove(r.diskPath(name)); err != nil {
```

Comment:
we need to make notExists error as not error.. but, here its going to return error for 'notExists' also..
same comment applies for next one also..